src/phoenix/server/api/schema.py

```diff
@@ -234,7 +234,10 @@ def delete_project(self, info: Info[Context, None], id: GlobalID) -> Query:
             return Query()
         type_name, node_id = from_global_id(str(id))
         if type_name != "Project":
-            return Query()
+            raise ValueError(
+                "The id passed to deleteProject must correspond to a node of type Project, "
+                f"but was passed a node of type: {type_name}"
+            )
```

Comment:
I would create a utility to do this `from_global_id_with_type(id, type_name)`

Reply:
addressed in aa99e13095fe96f152ba8dd98670353a59df912d